src/main/pg/pg.h

```diff
@@ -134,7 +134,7 @@ extern const uint8_t __pg_resetdata_end[];
 
 #define PG_REGISTER_WITH_RESET_FN(_type, _name, _pgn, _version)         \
     extern void pgResetFn_ ## _name(_type *);                           \
-    PG_REGISTER_I(_type, _name, _pgn, _version, .reset = {.fn = (pgResetFunc*)&pgResetFn_ ## _name }) \
+    PG_REGISTER_I(_type, _name, _pgn, _version, .reset = {.fn = (pgResetFunc*)(void (*)())&pgResetFn_ ## _name }) \
```

Comment:
There is actually bug in the code - pgResetFunc is declared to take length as second argument. This is unused in current code, it may be best to fix the declaration and call

Reply:
Good point. I assumed that it was used for `PG_REGISTER_ARRAY_WITH_RESET_FN()`, but then I did have to fix the cast there as well, so it isn't.

Reply:
I had another look into this, and we are actually supplying the size of the parameter group in the function call (https://github.com/betaflight/betaflight/blob/93f933f01facf0a2925a7a03320b2aa60ee9b913/src/main/pg/pg.c#L56) - there is just no reset function that is making use of it. Not sure if removing it is the right move, there might be uses for it in the future. Thoughts?

Reply:
This inconsistency shall be removed. Either by adding len parameter to all reset functions of by removing it in call.
Fixing it in pg.c has much smaller footprint, it can be added back if necessary.

Reply:
I guess so...

Reply:
Done.